packages/compass-aggregations/src/modules/stage.spec.js

```diff
@@ -310,20 +431,21 @@ describe('Stage module', () => {
       it('generates timestamp', () => {
         expect(generated.ts.low).to.equal(-321469502);
       });
+
       it('returns the stage string', () => {
         expect(generateStageAsString(stage)).to.deep.equal(`{$match: {
-  code: Code('some code'),
-  oid: ObjectId('5a7382114ec1f67ae445f778'),
-  bin: Binary('aakjadfjadfldksjfadf', '1'),
-  dbref: DBRef('db.coll', '1'),
-  nl: NumberLong('3'),
-  nd: NumberDecimal('5.00000001'),
-  ni: NumberInt('5'),
-  minkey: MinKey(),
-  maxkey: MaxKey(),
-  isodate: ISODate('1999-01-01'),
-  regexp: RegExp('/^[a-z0-9_-]{3,16}$/'),
-  ts: Timestamp(-321469502, 367)
+ code: Code('some code'),
+ oid: ObjectId('5a7382114ec1f67ae445f778'),
+ bin: BinData(1, 'YWFramFkZmphZGZsZGtzamZhZGY='),
+ dbref: DBRef('coll', '1'),
+ nl: NumberLong(3),
+ nd: NumberDecimal('5.00000001'),
+ ni: 5,
+ minkey: MinKey(),
+ maxkey: MaxKey(),
+ isodate: ISODate('1999-01-01T00:00:00.000Z'),
```

Comment:
And this?

Reply:
Hard to say but it looks like it is now more correct than before https://docs.mongodb.com/manual/reference/method/Date/

Reply:
Yes :+1: